packages/server/postgres/init.sql

```diff
@@ -0,0 +1,2 @@
+DROP DATABASE IF EXISTS development;
```

Comment:
if it's easier to use the same name for dev & prod, let's do it.
looking back, having `actionDevelopment` and `actionProduction` in rethinkdb was probably a mistake. plus they're really long names!